week-1-practise/level-1/8-function-output-1.js

```diff
@@ -1,12 +1,13 @@
 function getNumber() {
-    return Math.random() * 10;
+  return Math.random() * 10;
 }
 
 var result = getNumber();
 console.log(result);
 
 // Answer these questions:
 // - What does this program do?
-// - How many parameters does the function take?
-// - What is the function name?
-// - Where is the function called? with what parameters?
\ No newline at end of file
+//-returns a floating-point, pseudo-random number between range [0,1) , 0 (inclusive) and 1 (exclusive).
```

Comment:
It means that  Math.random() function returns a number that is greater or equal to 0 and less than 1.
And yes it is correct to say that  Math.random() produces numbers in the range 0 to 1, but getNumber() does not. 
because  :
Math.random()  will produce a number  greater or equal to 0 and less than 1
getNumber() will produce a number  greater or equal to 0 and less than 10